downstream/modules/platform/con_storage-options-for-operator-installation-on-ocp.adoc

```diff
@@ -0,0 +1,14 @@
+[id="con_storage-options-for-operator-installation-on-ocp_{context}"]
+= Storage options for {OperatorPlatform} installation on {OCP}
+
+If you are using file-based storage and your installation scenario includes {HubName}, ensure that you change the `ReadWriteOnce` default storage option for {OperatorPlatform} to `ReadWriteMany`.
+
+{HubNameStart} requires `ReadWriteMany` storage, Azure Blob storage, or an s3-compliant storage for operation because multiple pods responsible for upload and download of content (for example, collections) need access to the same content. Multiple pods can have access to the same persistent volume claim (PVC) as long as you select the `ReadWriteMany` storage option.
```

Comment:
```suggestion
{HubNameStart} requires `ReadWriteMany` storage, Azure Blob storage, or an s3-compliant storage for operation so that multiple pods can access shared content, such as collections.
```

Reply:
I don't think the 2nd sentence is necessary for minimalism. You're basically explaining RWX access, which is covered in k8s and OCP docs.

Reply:
Well, you have just cleared up the thing I was most confused about. My technological gap had me unsure about it, but since you're are knowledgeable about RWX, it clears things up. There were too many "or" conjunctions, which was muddying the waters, so to speak. Updating accordingly.